egs/material/s5/conf/lang/swahili.conf

```diff
@@ -0,0 +1,25 @@
+# speech corpora files location
+corpus=/path/to/corpus/
+# test audio files to decode
+audio_path_analysis1=/path/to/analysis1/audio/
```

Comment:
I think it would actually be clearer if you just used the paths on the CLSP grid rather than placeholders.  This will give the user more indication what is expected.  You can put a comment at the top of these files saying that the user is expected to replace these with values that work for their location.

Reply:
Ok, I'll update the paths to the form they originally looked like, that is the absolute paths on the CLSP grid.

Reply:
Done.